TFT/src/User/Configuration.h

```diff
@@ -998,13 +998,13 @@
  * Generic Debug
  * Uncomment/Enable to enable arbitrary debug serial communication to SERIAL_DEBUG_PORT defined in board specific Pin_xx.h file.
  */
-//#define DEBUG_SERIAL_GENERIC  // Default: commented (disabled)
+// #define DEBUG_SERIAL_GENERIC  // Default: commented (disabled)
```

Comment:
I suspect you committed this file by mistake

Reply:
Yes, by mistake. Will be fixed in an upcoming bugfix.